File: 657. Robot Return to Origin.cpp
class Solution {
public:
    bool judgeCircle(string moves) 
    {
        int start = 0, end = 0;
        for(char c:moves)
        {
            if(c == 'R') end++;
            else if(c == 'L')end--;
            else if(c == 'U') start--;
            else if(c == 'D') start++;
        }
        return start == 0 and end == 0;
    }
};

File: 657. Robot Return to Origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        start = 0
        for move in moves:
            if move == 'U':
                start += 1
            elif move == 'D':
                start -= 1
            
            if move == 'L':
                start -= 100
            elif move == 'R':
                start += 100
        return start == 0